cmd/notation/sign.go

```diff
@@ -90,7 +90,7 @@ Example - [Experimental] Sign an OCI artifact identified by a tag and referenced
 		Long:  longMessage,
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 0 {
-				return errors.New("missing reference")
+				return errors.New("missing reference to the artifact. Expecting <registry>/<repository>:<tag> or <registry>/<repository>@<digest>")
```

Comment:
Question. Now that we have the experimental flags which also take the OCI-layout, should we make this error message more inclusive to also describe the other parameters it can take? Or could we make it more generic and say "missing reference: use notation verify --help to see what parameters are required"

Reply:
Good idea. Updated the code.